spec/aruba/api_spec.rb

```diff
@@ -1071,6 +1071,48 @@ def actual_permissions
 
         expect(ENV[variable]).to eq '1'
       end
+
```

Comment:
I'm interested to see, if this #405 is fixed by your change. Would be quite happy if the answer is yes. Yet again, please move this to another PR.